content/source/docs/enterprise-beta/api/workspaces.html.md

```diff
@@ -71,30 +71,31 @@ $ curl \
          }
       },
       "links":{
-         "self":"/api/v2/organizations/skierkowski/workspaces/ws-2Qhk7LHgbMrm3grF"
+         "self":"/api/v2/organizations/my-organization/workspaces/ws-2Qhk7LHgbMrm3grF"
       }
    }
 }
 ```
 
 
 
-## Create a Workspace with VCS Configuration
+## Create a Workspace with a VCS Configuration
+
+The default `/workspaces` endpoint creates a workspace without configuring the VCS connection (`ingress-trigger`). In Beta there is no dedicated endpoint for managing the `ingress-trigger` so it must be configured at the time of workspace creation.
```

Comment:
You can now patch to this endpoint to update things (except you cannot update the repo that the workspace is linked to):

https://github.com/hashicorp/atlas/pull/5012

Reply:
I'll remove the line about managing `ingress-trigger` and the PATCH endpoint next week.